ql/src/test/queries/clientpositive/acid_insert_overwrite_update.q

```diff
@@ -26,7 +26,6 @@ insert overwrite table sequential_update values(current_timestamp, 0, current_ti
 delete from sequential_update where seq=2;
 select distinct IF(seq==0, 'LOOKS OKAY', 'BROKEN'), regexp_extract(INPUT__FILE__NAME, '.*/(.*)/[^/]*', 1) from sequential_update;
 
-alter table sequential_update compact 'major';
```

Comment:
Why change this?

Reply:
It turned out that the Q test running environment doesn't start the HMS background threads at all. As a result the issued compactions are never processed. From now on it is not possible to initiate a second compaction on the same table with the same write id before the previous one is cleaned up. As a result, the second compaction request was refused in this test. BTW this questions the necessity of the compaction commands in these tests for me, but I only removed only the second one  to make the test green again.